lib/elixir/lib/keyword.ex

```diff
@@ -867,6 +867,29 @@ defmodule Keyword do
     raise KeyError, key: key, term: original
   end
 
+  @doc """
+  Replaces the value under `key` using the given function only if
+  `key` already exists in `keywords`.
```

Comment:
```suggestion
  Replaces the value under `key` using the given function only if
  `key` already exists in `keywords`.
  
  In comparison to `replace/3`, this can be useful when it's expensive to calculate the value.
```